### ISSUE TITLE
Change the visibility of the fields of struct `Opcode` to public

### ISSUE THREAD

**Issue by hangj** [kromych/disarm64#2]
Hi, the fields of struct `Opcode` is private  

```rust
#[derive(Debug, PartialEq, Eq, Copy, Clone)]
pub struct Opcode {
    mnemonic: Mnemonic,
    operation: Operation,
}
```

I can't access `mnemonic` and `operation`, could you please change the visibility? Thanks!

**Comment by kromych** [comment]
Yep, will do!

**Comment by kromych** [comment]
Fixed in 0.1.19
